moped-editor/src/components/GridTable/GridTableNewItem.js

```diff
@@ -26,12 +25,16 @@ const GridTableExport = ({ query }) => {
             {query.config.new_item_label}
           </Button>
         ))}
+      {// add empty div of equal height if there is no button to match styles
+      !query.config.showNewItemButton && (
+        <div style={{ height: "36.5px" }}></div>
+      )}
```

Comment:
if we arent going to render a button, then render an empty div of button size. 

I considered conditionally adding padding to the div in GridTableSearch, but this is the component that knows whether or not it is going to render the button.